### PR TITLE
Add data in Badge Table

### PR DESCRIPTION
**Pull request by AnneDAMIEN**

### CODE REVIEW

File: server/database/data/badges.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+    {
+    "name": "Maître des recettes",
+    "description": "Tu as posté plus de 10 recettes, félicitations",
+    "image": "/images/Badge_id1.png"
```
Comment (ioayoub) [inline server/database/data/badges.json:5]: your badge images aren't in the declared folder.

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: server/database/schema.sql [hunk 2]
```diff
@@ -15,7 +15,8 @@ CREATE TABLE ingredient (
 CREATE TABLE badge (
```
[the remark this answers is not in the document]
Comment (AnneDAMIEN) [reply server/database/schema.sql]: I approved Angelique's work so it's done now

File: server/database/data/badges.json (hunk 1 above)
Comment (AnneDAMIEN) [reply server/database/data/badges.json:5]: It's done

**Review by yannduhamel — DISMISSED** [review]

**Review by enerisan — APPROVED** [review]

**Review by angelique888 — APPROVED** [review]

File: server/database/fixtures/BadgeSeeder.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const AbstractSeeder = require("./AbstractSeeder");
+
+class BadgeSeeder extends AbstractSeeder {
+  constructor() {
+    super({ table: "badge", truncate: true });
+  }
+
+  run() {
+    for (let i = 0; i < badges.length; i += 1) {
+      const ingredient = badges[i];
```
Comment (ioayoub) [inline server/database/fixtures/BadgeSeeder.js:12]: ingredient or badge ? :)

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

File: server/database/fixtures/BadgeSeeder.js (hunk 3 above)
Comment (AnneDAMIEN) [reply server/database/fixtures/BadgeSeeder.js]: Badge, of course :)

**Review by yannduhamel — APPROVED** [review]

**Review by angelique888 — APPROVED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by enerisan — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**